bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_TaskScheduler_doc.h

```diff
@@ -0,0 +1,40 @@
+/******************************************************************************
+*                              SofaPython3 plugin                             *
+*                  (c) 2021 CNRS, University of Lille, INRIA                  *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU Lesser General Public License as published by    *
+* the Free Software Foundation; either version 2.1 of the License, or (at     *
+* your option) any later version.                                             *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License *
+* for more details.                                                           *
+*                                                                             *
+* You should have received a copy of the GNU Lesser General Public License    *
+* along with this program. If not, see <http://www.gnu.org/licenses/>.        *
+*******************************************************************************
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+
+#pragma once
+
+namespace sofapython3::doc::taskscheduler
+{
+
+static auto init =
+    R"(
+    Initialize the main task scheduler with a number of threads.
+    )";
+
+static auto getThreadCount =
+    R"(
+    Returns the number of threads used by the main task scheduler.
+    )";
+
+static auto GetHardwareThreadsCount =
```

Comment:
not consistent method naming: getThreadCount and GetHardwareThreadsCount
g vs G

Reply:
This is because it uses the same convention than the function names. `getThreadCount` is not static while `GetHardwareThreadsCount` is. Do you also have a problem with function names, or just with those strings?

Reply:
I don't want any problem...
let's keep it asis